compose-table/src/androidTest/java/org/dhis2/composetable/data/TableInputUiData.kt

```diff
@@ -0,0 +1,50 @@
+package org.dhis2.composetable.data
+
+import org.dhis2.composetable.model.RowHeader
+import org.dhis2.composetable.model.TableCell
+import org.dhis2.composetable.model.TableHeader
+import org.dhis2.composetable.model.TableHeaderCell
+import org.dhis2.composetable.model.TableHeaderRow
+import org.dhis2.composetable.model.TableModel
+import org.dhis2.composetable.model.TableRowModel
+
+val tableData = listOf(
```

Comment:
how this would look like?
          |      A        |         B      |
          |  1 | 2 | 3  |    1 | 2 | 3  |
row 1 | 00| 0 | 0  |  12 |    |     |
row 2 |     |    |     |       |    |     |

Reply:
| A             | B           |
          | 1   | 2  | 3 | 1 | 2 | 3 |
row 1 | 12 |     |    |    |    |    |
row 2 |      |     |    |    |    |    |